tests/cli/test_rasa_train.py

```diff
@@ -558,3 +558,24 @@ def test_get_valid_config(parameters):
 def test_get_valid_config_with_non_existing_file():
     with pytest.raises(SystemExit):
         _get_valid_config("non-existing-file.yml", CONFIG_MANDATORY_KEYS)
+
+
+def test_train_nlu_finetune_with_model(
+    run_in_simple_project_with_model: Callable[..., RunResult]
+):
+    temp_dir = os.getcwd()
+
+    assert os.path.exists(os.path.join(temp_dir, "models"))
+    files = rasa.shared.utils.io.list_files(os.path.join(temp_dir, "models"))
+    assert len(files) == 1
+
+    output = run_in_simple_project_with_model("train", "nlu", "--finetune")
+
+    assert [
+        s
+        for s in output.outlines
+        if "Loading NLU model from models/"
```

Comment:
i think it might be `models\` in Windows. In my opinion fine to do do something like:


```python
assert any(os.path.basename(files[0]) in line for line in output.outlines)
```

Reply:
I think skipping the test for Windows is nothing we should do as we need to ensure the same behavior on Windows

Reply:
That is so accurate!!! I forgot about the pathnames difference. I will try that. Thank YOU!

Reply:
I changed it and now it sets the directory separator that finds in the file path (so if it's ubuntu it will take the slash and if it's windows it will find and take the backslash). Let's see if it works. Otherwise I will use your assert statement.

Reply:
I also tried your assert statement but it fails again with error message :
```
Training NLU model...
No NLU model for finetuning found. Please make sure to either specify a path to a previous model or to have a finetunable model within the directory 'models'.
```